pkg/server/server.go

```diff
@@ -159,12 +167,65 @@ func createEnvironment(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	//space delimited annotation of valid hostnames
+	var hostsList bytes.Buffer
+
+	//Loop through slice of HostNames
+	for index, value := range tempJSON.HostNames {
+		//Verify each Hostname matches regex
+		validIP := validIPAddressRegex.MatchString(value)
+		validHost := validHostnameRegex.MatchString(value)
+
+		if !(validIP || validHost) == true {
```

Comment:
Why not `if validIP || validHost {`?  Wrapping in parentheses and then checking against an explicit boolean isn't necessary.